src/core/lib/event_engine/thread_pool.cc

```diff
@@ -89,7 +98,7 @@ bool ThreadPool::Queue::Step() {
 ThreadPool::ThreadPool(int reserve_threads)
     : reserve_threads_(reserve_threads) {
   for (int i = 0; i < reserve_threads; i++) {
-    StartThread(state_);
+    StartThread(state_, false);
```

Comment:
Suggest `StartThread(state_, /*throttled=*/false);` to make it self-documenting.

Reply:
sg... I'm a bit used to letting my editor do this for me :)

Reply:
Thanks. The clang tidy [bugprone-argument-comment](https://clang.llvm.org/extra/clang-tidy/checks/bugprone/argument-comment.html) check can help in the long run if the argument's name or meaning ever changes.